tests/plugins/test_manager.py

```diff
@@ -85,13 +85,24 @@ def test_load_plugins_error(plugin_manager, mocker):
         assert "load_plugins error" in str(exc.value)
 
 
-def test_load_setuptools_entrypoints_error(plugin_manager, mocker):
+def test_load_setuptools_entrypoints_valueerror(plugin_manager, mocker):
     with mocker.patch(
         "pluggy._manager.importlib_metadata.distributions",
-        side_effect=ValueError("load_setuptools_entrypoints error"),
+        side_effect=ValueError("load_setuptools_entrypoints ValueError"),
     ):
         with pytest.raises(PluginError) as exc:
             plugin_manager.load_setuptools_entrypoints("conda")
         assert plugin_manager.get_plugins() == set()
         assert exc.value.return_code == 1
-        assert "load_setuptools_entrypoints error" in str(exc.value)
+        assert "load_setuptools_entrypoints ValueError" in str(exc.value)
+
+
+def test_load_setuptools_entrypoints_importerror(caplog, plugin_manager, mocker):
+    with mocker.patch(
+        "pluggy._manager.importlib_metadata.distributions",
+        side_effect=ImportError("load_setuptools_entrypoints ImportError"),
+    ):
+        caplog.set_level(logging.INFO, logger="conda.plugins.manager")
+        plugin_manager.load_setuptools_entrypoints("conda")
+        assert plugin_manager.get_plugins() == set()
+        assert "load_setuptools_entrypoints ImportError" in caplog.text
```

Comment:
I bet you could add an `ImportError` in a different way. See https://github.com/conda/conda/blob/main/tests/gateways/test_repodata_gateway.py#L113 or https://github.com/conda/conda/blob/196148c25db8e5aa2a78124ce2232818d6316fc3/tests/test_deprecations.py#L170